dask/multiprocessing.py

```diff
@@ -191,10 +195,18 @@ def get(
             os.environ["PYTHONHASHSEED"] = "6640"
         context = get_context()
         pool = ProcessPoolExecutor(
-            num_workers, mp_context=context, initializer=initialize_worker_process
+            num_workers,
+            mp_context=context,
+            initializer=initializer or initialize_worker_process,
         )
         cleanup = True
     else:
+        if initializer is not None:
+            raise ValueError(
+                "Incompatible values: ``pool`` and ``initializer`` may not "
+                "both be set at the same time.  At least one of them must be "
+                "set to ``None``."
+            )
```

Comment:
Not sure if we should raise an error as someone could have this set in the `dask.config` already. Maybe we should be warning instead?

```suggestion
    else:
        if initializer is not None:
            warn(
                "The ``initializer`` argument is ignored when ``pool`` is provided."
                " The user should configure ``pool`` with the needed ``initializer`` on creation."
            )
```

Reply:
Agreed.